ls1intum/Artemis
# `Iris`: Add Iris logo

### Checklist
#### General


#### Client

### Motivation and Context


To make Iris more personal, we want to add a logo for it.

### Description

@LordKulkan graciously created a logo for us, which we implement in this PR.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Programming exercise with Iris enabled

1. Open the chat window in the programming exercise -> Logo as in screenshot
2. Click on the "i" button
3. See the logo on the about iris page as in the screenshot 

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests

### Test Coverage
Not possible due to failures on develop-iris.

### Screenshots


About Iris page:
<img width="1679" alt="grafik" src="https://github.com/ls1intum/Artemis/assets/18218285/fcf7f1aa-e259-4dc1-a7ff-408a55c983df">

Chat window:
<img width="363" alt="grafik" src="https://github.com/ls1intum/Artemis/assets/18218285/52c29a1d-2cc1-4b30-b6c1-485527d4ab76">
